cvat/apps/engine/serializers.py

```diff
@@ -1080,8 +1086,9 @@ def update(self, instance, validated_data):
             'specific_attributes': parse_specific_attributes(instance.specific_attributes)
         }
         storage = get_cloud_storage_instance(cloud_provider=instance.provider_type, **details)
+        is_testing = os.environ.get('TESTING', False)
```

Comment:
@Marishka17 , I don't think that we need to have the check in production code. Need to find another idea.